book/website/ethical-research/data-hazards/dh-how-to-use.md

```diff
@@ -0,0 +1,113 @@
+3. Talking about them with other people.
+
+4. [Printing out label cards](https://datahazards.com/_downloads/b92f884790471e61048c5e0fee4dd08e/DataHazards_PrintableCards.pdf) and using them in a workshop.
+
+(er-datahazardshowtouse-apply)=
+## **2. Applying** the Data Hazard labels 
+Applying the labels means deciding which labels are relevant to a project.
+There is no prescriptive way to do this. However, we suggest one way to go through each label one at a time and decide why or why not it applies to your project.
+
+- **What projects can I apply the labels to?** 
+
+We recommend that you apply the labels to your own work unless you've been invited to give feedback on another piece of work.
+
+The labels have been applied to many different types of projects that use data or data-intensive methods.
+Some examples of projects include:
+
+- A predictive model that aims to use machine learning to predict human traits from their genetic mutations.
+
+- A digital humanities project that uses web-scraping and natural language processing to analyse the text of political speeches.
+
+- An NHS data-linkage project to create a new database for researchers to use.
+
+- A modelling project that aims to improve whole-cell models of a bacteria.
+
+Labelling these projects was part of various workshops and is not recorded or shared publicly.
+
+- **When should I apply Data Hazards in my work?** 
+
+Data Hazards labels can be applied at any stage in the research life-cycle.
+Ideally, the best time to apply the labels is close to the beginning of the research project, at the same time as you might consider your university ethics process, or {ref}`cm-dif-articles-registered-reports` (you can also pre-register your Data Hazards analysis!). 
+However, much of our research builds on itself, so researchers have also found [workshops](https://datahazards.com/contents/materials/workshop.html) useful to reflect and plan what they can do better in follow-up projects.
+
+At the beginning of your workflow, you might want to prepare to avoid certain Data Hazards if you can, and if you can't avoid them because of where your data has come from, you may want to acknowledge this. For example, if you have a [sensitive data project](https://the-turing-way.netlify.app/project-design/sdp.html), what Data Hazard labels will apply, and/or what can you do to design your project in a way that avoids certain harms?
+ 
+As you are collecting and analyzing your data, you might want to iteratively think of the potential Data Hazards that exist in the information you are actively collecting. 
+If you have a project where data has already been collected in the past, you can still apply and think of what labels may be relevant to the dataset. 
+
+When you are reporting your results, it is recommended you also think of reporting mitigation strategies together with the labels; examples of how others have done this can be found [here](https://datahazards.com/contents/materials/examples.html). 
```

Comment:
Great point - and I love that you're linking to the examples. Best practice for linking is to write out what you're linking to, instead of linking the word "here", so for example:

```
the Data Hazards website contains [examples of how others have done this](https://datahazards.com/contents/materials/examples.html)
```